index.html

```diff
@@ -216,6 +239,7 @@ <h2><code>PointerEvent</code> interface</h2>
     boolean     isPrimary = false;
     sequence&lt;PointerEvent> coalescedEvents = [];
     sequence&lt;PointerEvent> predictedEvents = [];
+    DeviceProperties deviceProperties;
```

Comment:
Should you have idl definition for DeviceProperties before PointerEventInit?

Reply:
I've moved it one section up, but I've kept it as its own section. I'm using [eventTarget](https://dom.spec.whatwg.org/#interface-eventtarget) as an example. WDYT?